src/style.css

```diff
@@ -0,0 +1,29 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.star {
+  background: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
```

Comment:
Please remove margin-left for each first star
![image](https://github.com/mate-academy/layout_stars/assets/54936044/44b094f7-861e-46e5-984e-93cd1dd6b39b)